aries-site/src/layouts/main/Header.js

```diff
@@ -47,6 +48,7 @@ const StyledHeader = ({ ...rest }) => {
             icon={<SearchIcon />}
             onClick={() => setSearchFocused(true)}
           />
+          <ThemeModeToggle />
```

Comment:
When I open the search, the theme toggle disappears and the search shifts down. it feels like it'd be fine to let the theme toggle stay where it is, and the search input opens to the left